### ISSUE TITLE
Add support for BitMex exchange. 

### ISSUE THREAD

**Issue by Derivatives2ProgrammingLangueages** [garethdmm/gryphon#67]
Could someone add support for the BitMex exchange? And if someone had the documentation to add a new exchange direct me to that but still keep the issue open?

**Comment by garethdmm** [comment]
Yeah this is on the roadmap for sure. If you want to get started on it yourself here's some quick tips. The outline for implementing a new exchange is roughly this:
1) Implement the exchange trading interface all exchange wrappers use: https://gryphon.readthedocs.io/en/latest/exchange_integrations.html#exchange-trading-interface
2) Add it to gryphon.lib.exchange.exchange_factory, this is pretty straightforward if you look at what happens for other exchanges
3) Add it to the unit tests

An example would be this recent pull request which does a lot right: https://github.com/garethdmm/gryphon/pull/66
